packages/logger/src/Logger.ts

```diff
@@ -1,117 +1,88 @@
+ * const logger = new Logger({ serviceName: 'serverlessAirline' });
  *
- * If you use function-based Lambda handlers you can use the {@link injectLambdaContext} middleware
- * to automatically add context to your Lambda logs.
- *
- * @example
- * ```typescript
- * import { Logger } from '@aws-lambda-powertools/logger';
- * import { injectLambdaContext } from '@aws-lambda-powertools/logger/middleware';
- * import middy from '@middy/core';
- *
- * const logger = new Logger();
- *
- * const lambdaHandler = async (_event: unknown, _context: unknown) => {
- *     logger.info('This is an INFO log with some context');
+ * export const handler = async (event, context) => {
+ *   logger.info('This is an INFO log');
+ *   logger.warn('This is a WARNING log');
  * };
- *
- * export const handler = middy(lambdaHandler).use(injectLambdaContext(logger));
  * ```
  *
- * ### Object oriented usage with decorators
- *
- * If instead you use TypeScript classes to wrap your Lambda handler you can use the {@link injectLambdaContext} decorator.
+ * To enrich the log items with information from the Lambda context, you can use the {@link Logger.addContext | addContext()} method.
  *
  * @example
  * ```typescript
  * import { Logger } from '@aws-lambda-powertools/logger';
- * import type { LambdaInterface } from '@aws-lambda-powertools/commons/types';
  *
- * const logger = new Logger();
+ * const logger = new Logger({ serviceName: 'serverlessAirline' });
  *
- * class Lambda implements LambdaInterface {
- *     // Decorate your handler class method
- *     ⁣@logger.injectLambdaContext()
- *     public async handler(_event: unknown, _context: unknown): Promise<void> {
- *         logger.info('This is an INFO log with some context');
- *     }
- * }
+ * export const handler = async (event, context) => {
+ *   logger.addContext(context);
  *
- * const handlerClass = new Lambda();
- * export const handler = handlerClass.handler.bind(handlerClass);
+ *   logger.info('This is an INFO log with some context');
+ * };
  * ```
  *
- * ### Functions usage with manual instrumentation
- *
- * If you prefer to manually instrument your Lambda handler you can use the methods in the Logger class directly.
+ * You can also add additional attributes to all log items using the {@link Logger.appendKeys | appendKeys()} method.
  *
  * @example
  * ```typescript
- * import { Logger } from '@aws-lambda-powertools/logger';
+ * export const handler = async (event, context) => {
+ *   logger.appendKeys({ key1: 'value1' });
  *
- * const logger = new Logger();
+ *   logger.info('This is an INFO log with additional keys');
  *
- * export const handler = async (_event, context) => {
- *     logger.addContext(context);
- *     logger.info('This is an INFO log with some context');
+ *   logger.removeKeys(['key1']);
  * };
- * ```
+ *```
+ *
+ * If you write your functions as classes and use TypeScript, you can use the {@link Logger.injectLambdaContext} class method decorator
```

Comment:
functions are Lambda functions or standalone methods?

Reply:
What I mean by this is:

```ts
class Lambda {
  public handler(event, context) { // <- this is the lambda handler
     // ...
  }
}
```